documentation/LLC-quickstart.md

```diff
@@ -0,0 +1,401 @@
+Getting Started - Generate the RLC rest-level client libraries.
+================================================================
+# Prerequisites
+You may refer to this [link](https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md#prerequisites) for the environment set up prerequisites in azure-sdk-for-js repository. 
+
+# How to generate RLC
+We are working on to automatically generate everything right now, but currently we still need some manual work to get a releasable package. Here're the steps of how to get the package.
+1. **Create a swagger/README.md file.**  
+    We are using autorest to generate the code, but there's a lot of command options and in order to make the regenerate process easier in the cases of refresh the rest api input or change the code generator version, you need to document the generate command parameters.  
+    Here's an example of the swagger/README.md
+    ~~~
+    
+    # Azure Purview Catalog TypeScript Protocol Layer
```

Comment:
It looks like we are not getting the rollup file generated. I think it is okay, you can just mention that manually need to create a `rollup.config.js` file in the project folder with the following content

```js
import { makeConfig } from "@azure/dev-tool/shared-config/rollup";

export default makeConfig(require("./package.json"));

```

Reply:
Updated.